ghost-cat-app/src/app/View/search-page/search-page.component.ts

```diff
@@ -64,6 +68,7 @@ export class SearchPageComponent implements OnInit {
   }
 
   public updateCameraSite(cameraSite: CameraLocation): void {
-    this.cameraTrap = cameraSite.label;
```

Comment:
I think it will work if you update this method to be:
`if (this.cameraTrapsSelected.includes(cameraSite.label) {`
    `this.cameraTrapsSelected = this.cameraTrapsSelected.filer(trap => trap != cameraSite.label);`
`} else {`
    `this.cameraTrapsSelected.push(cameraSite.label);`
`}`